var/spack/repos/builtin/packages/vep-cache/package.py

```diff
@@ -3,72 +3,38 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-import itertools
-
 from spack.package import *
 
 
-def _vep_cache_filename(version, species, source, assembly):
-    assembly_source = "vep" if source == "ensembl" else f"{source}_vep"
-    return f"{species}_{assembly_source}_{version}_{assembly}.tar.gz"
-
-
-def _vep_cache_resource(version, species, source, assembly, indexed, dest=""):
-    filename = _vep_cache_filename(version, species, source, assembly)
-    dir_name = "indexed_vep_cache" if indexed else "vep"
-    root = f"https://ftp.ensembl.org/pub/release-{version}/variation/{dir_name}"
-    url = join_path(root, filename)
-    when = [
-        f"@{version}",
-        "~use_vep_installer",  # Only need these resources when we don't use the installer
-        "+indexed" if indexed else "~indexed",  # Only need the appropriate indexed versiona
-        f"assembly_source={source}",  # Only reference the assembly source defined here
-        f"species={species}",  # Only need the requested species
-        # We only need to match the specified assembly for human assemblies
-        f"assembly={assembly.lower()}" if species == "homo_sapiens" else "",
-    ]
-
-    kwargs = {
-        "name": filename,
-        "url": url,
-        "when": " ".join(when),
-        "destination": dest,
-        "expand": False,  # We'll expand this where it needs to go later
-    }
-
-    return resource(**kwargs)
-
-
 class VepCache(Package):
     """Separate installation and management for the Ensembl Variant Effect Predictor (vep)"""
 
     homepage = "https://useast.ensembl.org/info/docs/tools/vep/index.html"
-    has_code = False
     maintainers("teaguesterling")
+    # This is a dummy value to get spack to resolve resources, which are not downloaded
```

Comment:
I don't understand the comment.
There are no more resources - do we still need this?

Reply:
Going to merge this now, this is not terribly important.